src/libraries/System.Composition.Convention/src/PACKAGE.md

```diff
@@ -0,0 +1,96 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+`System.Composition.Convention` is part of the Managed Extensibility Framework (MEF) 2.0, a composition library for .NET that enables dependency injection through attributes.
```

Comment:
Actually this one is awkward to mention attributes here.  This demonstrates usage without attributes.  Should we update all of these to be more broad to indicate the other means of injection? 
```suggestion
`System.Composition.Convention` is part of the Managed Extensibility Framework (MEF) 2.0, a composition library for .NET that enables dependency injection through attributes or conventions.
```

Reply:
Ah, that might be better indeed